server/src/main/java/io/druid/curator/CuratorConfig.java

```diff
@@ -66,4 +69,14 @@ public void setEnableCompression(Boolean enableCompression)
   {
     this.enableCompression = enableCompression;
   }
+
+  public boolean isEnableAcl()
+  {
+    return enableAcl;
+  }
+
+  public void setEnableAcl(Boolean enableAcl)
```

Comment:
@drcrallen if someone typos a config, you want an error thrown versus a default value set